Makefile

```diff
@@ -63,6 +63,12 @@ statusgo-ios-simulator: xgo	##@cross-compile Build status-go for iOS Simulator
 	$(GOPATH)/bin/xgo --image farazdagi/xgo-ios-simulator --go=$(GO) -out statusgo --dest=$(GOBIN) --targets=ios-9.3/framework -v $(shell build/testnet-flags.sh) ./lib
 	@echo "iOS framework cross compilation done."
 
+statusgo-library: ##@cross-compile Build status-go as static library for current platform
+	@echo "Building static library..."
+	go build -buildmode=c-archive -o $(GOBIN)/libstatus.a ./lib
```

Comment:
I run a build and this is the output:
```
$ npm run rebuild

> status-nodejs@1.0.0 rebuild /Users/adamb/repos/status-nodejs
> node-gyp rebuild

  CXX(target) Release/obj.target/binding/src/status.o
  SOLINK_MODULE(target) Release/binding.node
ld: warning: object file (/Users/adamb/repos/status-nodejs/bin/libstatus.a(000000.o)) was built for newer OSX version (10.13) than being linked (10.7)
ld: warning: object file (/Users/adamb/repos/status-nodejs/bin/libstatus.a(000001.o)) was built for newer OSX version (10.12) than being linked (10.7)
ld: warning: object file (/Users/adamb/repos/status-nodejs/bin/libstatus.a(000006.o)) was built for newer OSX version (10.12) than being linked (10.7)
ld: warning: object file (/Users/adamb/repos/status-nodejs/bin/libstatus.a(000007.o)) was built for newer OSX version (10.12) than being linked (10.7)
ld: warning: object file (/Users/adamb/repos/status-nodejs/bin/libstatus.a(000003.o)) was built for newer OSX version (10.12) than being linked (10.7)
ld: warning: object file (/Users/adamb/repos/status-nodejs/bin/libstatus.a(000008.o)) was built for newer OSX version (10.12) than being linked (10.7)
ld: warning: object file (/Users/adamb/repos/status-nodejs/bin/libstatus.a(000005.o)) was built for newer OSX version (10.12) than being linked (10.7)
ld: warning: object file (/Users/adamb/repos/status-nodejs/bin/libstatus.a(000002.o)) was built for newer OSX version (10.13) than being linked (10.7)
ld: warning: object file (/Users/adamb/repos/status-nodejs/bin/libstatus.a(000004.o)) was built for newer OSX version (10.12) than being linked (10.7)
```
That's why I used `CGO_FLAGS=-mmacosx-version-min=10.7 CGO_LDFLAGS=-mmacosx-version-min=10.7 go build -installsuffix status -ldflags -v -buildmode=c-archive -o libstatus.a ./lib`.

Reply:
I actually have the same output even with these flags. You don't?

Reply:
Have you tried with `-installsuffix status` or some other value than `status`? It happens when it reuses cached libraries built without the flags.

Reply:
With `installsuffix` option I have a lot of similar warnings during `go build` as well as after `npm install`.

Reply:
Considering that it's just warning message, I propose to merge it (so other devs can use make to build a lib) and study how to get rid of those warnings in a separate PR.

Reply:
Hm. We can leave it for now and see how it's gonna work in clean environments when I finish https://github.com/status-im/status-nodejs/issues/5.